JASP-Desktop/widgets/jaspcontrolwrapper.cpp

```diff
@@ -137,6 +137,7 @@ AnalysisForm *JASPControlWrapper::form() const
 
 void JASPControlWrapper::addControlError(const QString &error)
 {
+	Log::log() << "Error: " << error << std::endl;
```

Comment:
Could you change "Error: " to "JASPControlWrapper::addControlError: ", otherwise it will be pretty much impossible to guess what kind of error it is from the log only...